.github/workflows/bors.yml

```diff
@@ -239,7 +239,7 @@ jobs:
           # storing and transferring oleans over the network.
           # Hopefully a future re-implementation of `cache` will obviate the present need for this hack.
           lake exe cache get Archive.lean
-          lake build Archive | tee --append stdout.log
+          bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build Archive | tee --append stdout.log"
```

Comment:
I guess in principle we should run the problem-matcher here, but we probably shouldn't give it our secrets